sapbtp-operator-charts/templates/crd.yml

```diff
@@ -880,6 +889,9 @@ spec:
                 description: The name of the service plan
                 minLength: 1
                 type: string
+              shared:
+                description: Indicates if the instances is shared
```

Comment:
ua review

Reply:
Whether the instance is shared. Default value is false.